api-docs/openapi/content/cloud/security-schemes.yml

```diff
@@ -0,0 +1,56 @@
+TokenAuthentication:
+  type: http
+  scheme: token
+  bearerFormat: InfluxDB Token String
+  description: |
+    ### Token authentication scheme
+
+    InfluxDB API tokens ensure secure interaction between users and data. A token belongs to an organization and identifies InfluxDB permissions within the organization.
+
+    Include your API token in an `Authentication: Token YOUR_API_TOKEN` HTTP header with each request.
+
+    ### Example
+
+    `curl https://us-east-1-1.aws.cloud2.influxdata.com/
+          --header "Authentication: Token YOUR_API_TOKEN"`
+
+    For more information and examples, see the following:
+      - [Use tokens in API requests](https://docs.influxdata.com/influxdb/cloud/api-guide/api_intro/#authentication).
```

Comment:
> re: links and section headers, think it may be helpful to match? See API Quick Start, Bootstrap your application, and Authorization, but not explicitly Use tokens in API requests, which made me wonder...for good practice, to keep headings in a procedure imperative, so Authorization >> Authorize API requests

#3350